src/kudu/security/security_flags.cc

```diff
@@ -27,14 +27,14 @@ namespace security {
 // below. The DH AES ciphers are not included since we are not configured to
 // use DH key agreement.
 const char* const SecurityDefaults::SecurityDefaults::kDefaultTlsCiphers =
+                                   "TLS_AES_256_GCM_SHA384:"
+                                   "TLS_CHACHA20_POLY1305_SHA256:"
+                                   "TLS_AES_128_GCM_SHA256:"
+                                   "TLS_AES_128_CCM_8_SHA256:"
+                                   "TLS_AES_128_CCM_SHA256:"
                                    "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:"
                                    "ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:"
-                                   "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:"
-                                   "ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:"
-                                   "ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:"
-                                   "AES256-GCM-SHA384:AES128-GCM-SHA256:"
-                                   "AES256-SHA256:AES128-SHA256:"
-                                   "AES256-SHA:AES128-SHA";
+                                   "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256";
```

Comment:
neither is this one installed
ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256